releng/tools/org.openjdk.jmc.util.releasenotes/notes/8.2/notes.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<?xml-stylesheet type="text/xsl" href="stylesheet.xsl"?>
+<notes>
+	<component>
+		<!-- ******************************* GENERAL ****************************************** -->
+		<name>General</name>
+		<note version="8.2">
+			<title>JMC 8.2 - New Release!</title>
+			<description>
+				This is a new minor release of Java Mission Control. The
+				JMC application requires JDK 11+ to run, but can still be used to
+				connect to, and parse JFR recordings from, OpenJDK 8u272+ and Oracle
+				JDK 7u40+. It can also still open and visualize flight recordings
+				from JDK 7 and 8.
+			</description>
+			<image>general/jmc.png</image>
+		</note>
+		<note version="8.2">
+			<title>Eclipse 4.22 support</title>
+			<description>
+				The Mission Control client is now built to run optimally
+				on Eclipse 2021-06 and later. To install Java Mission Control into
+				Eclipse, go to the update site (Help | Install New Software...). The
+				URL to the update site will be vendor specific, and some vendors
+				will instead provide an archive with the update site.
+			</description>
+			<image>general/eclipse.png</image>
+		</note>
+		<note version="8.2">
+			<title>Minor bugfixes and improvements</title>
+			<description>
+				There are 83 fixes and improvements in this release.
+				Check out the JMC 8.2 Result Dashboard
+				(https://bugs.openjdk.java.net/secure/Dashboard.jspa?selectPageId=20804)
+				for more information.
+			</description>
+			<image>general/issues.png</image>
+		</note>
+		<note version="8.2">
+			<title>Binary build for Apple ARM</title>
+			<description>
+				JDK Mission Control is now built for Apple ARM, allowing
+				JMC to be
+				run natively (without Rosetta x86 emulation) on Apple M1.
+			</description>
+			<image>general/apple.png</image>
+		</note>
+	</component>
+	<component>
+		<!-- ******************************* JMC Core ************************** -->
+		<name>Core</name>
+		<note version="8.2">
+			<title>Parser support for async profiler</title>
+			<description>
+				Parser support has been added for frame types generated
+				by async profiler, such as Native, C++ and Kernel.
+			</description>
+			<image>special/noimage.png</image>
+		</note>
+		<note version="8.2">
+			<title>System.gc() rule</title>
+			<description>
+				There is now a new rule for explicit invocations of
+				System.gc().
+			</description>
+			<image>core/dukegc.png</image>
+		</note>
+	</component>
+	<component>
+		<!-- ******************************* JFR ******************************* -->
+		<name>Java Flight Recorder (JFR)</name>
+		<note version="8.2">
+			<title>Heat map view</title>
+			<description>
+				A new heat map view has been added, which is handed to
```

Comment:
Not sure if "handed" is the right word to use here?

Reply:
"used"?